packages/libs/deer/README.md

```diff
@@ -0,0 +1,162 @@
+# deer
```

Comment:
In markdown, we use softwraps instead of hardbreaks, so we basically have one line per paragraph. Could you adjust this file please?

Reply:
fixed in [e0bfcd6](https://github.com/hashintel/hash/pull/1163/commits/e0bfcd65d688297910bb00fcb76911a517a4804c)